collect_app/src/test/java/org/odk/collect/android/widgets/AnnotateWidgetTest.java

```diff
@@ -123,45 +117,6 @@ public void whenReadOnlyOverrideOptionIsUsed_shouldAllClickableElementsBeDisable
         assertThat(getSpyWidget().annotateButton.getVisibility(), is(View.GONE));
     }
 
-    @Test
-    public void whenPromptHasDefaultAnswer_showsInImageView() throws Exception {
```

Comment:
Why did you remove these tests?

Reply:
So these tests were relevant when the answer images were being loaded synchronously via code, now that we are using Glide to load the image into imageView, I am not sure if we need them.

Reply:
So these tests were relevant when the answer images were being loaded synchronously via code, now that we are using Glide to load the image into imageView, I am not sure if we need them.

Reply:
Those tests were added to verify that adding [default answers](https://docs.getodk.org/form-logic/#static-defaults) (even like in this case default media answers works well) and the fact that we use Glide or not has nothing to do here. So I think we should keep them.

Reply:
I reverted that change with using Glide for now and added the tests back.